### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pvizner**
Added more background info and Spanish translation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+
+______________________________
+
+# Bienvenido a los Documentos del Video Kit de APEX!
+
+Estos documentos son un anexo del video kit móvil creado para el Audiovisual Preservation Exchange Program (APEX) de la New York University (NYU). Este kit incluye la cantidad mínima de equipamiento necesario para demostrar la digitalización de video análogo - excluyendo la máquina reproductora y el monitor CRT - y está diseñado para caber en una valija.
+
+Puede ver más sobre el programa APEX aquí: www.apexnyu.wordpress.com
+
+En esta documentacin encontrará:
```
Comment (retokromer) [inline README.md]: documentación?

Comment (pvizner) [reply README.md]: oh, sí!

Comment (retokromer) [reply README.md]: Ho contrabbandato dall’italiano… ;-)

**Comment by savcampbell** [comment]
Looks good to me!

**Review by jfarbowitz — APPROVED** [review]

**Pull request merged**